errors/common.go

```diff
@@ -44,6 +45,14 @@ func IsInvalidFormatErr(err error) bool {
 	return IsSameError(errInvalidFormat, err)
 }
 
+func ErrUnknownModule(mod string) TMError {
```

Comment:
Makes sense - I guess let's just work to migrate towards `basecoin/errors` to keep things clean